appveyor.yml

```diff
@@ -0,0 +1,15 @@
+version: '{build}'
+build_script:
+- cmd: >-
+    set RUBY_VERSION=200
+
+    set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+
+    gem install bundler
+
+    bundle install
+
+    build.netstd.cmd
+test: off
+on_finish:
+    - ps: $root = Resolve-Path .\artifacts; [IO.Directory]::GetFiles($root.Path, '*.*', 'AllDirectories') | % { Push-AppveyorArtifact $_ -FileName $_.Substring($root.Path.Length + 1) -DeploymentName to-publish }

```

Comment:
IIRC we instruct MSBuild to log at a highly summarised level to the
console, and at a detailed level to the file system.

On 19 Aug 2016 15:34, "Thomas Levesque" <notifications@github.com> wrote:

> In appveyor.yml
> <https://github.com/FakeItEasy/FakeItEasy/pull/821#discussion_r75481834>:
>
> > @@ -0,0 +1,15 @@
> > +version: '{build}'
> > +build_script:
> > +- cmd: >-
> > +    set RUBY_VERSION=200
> > +
> > +    set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
> > +
> > +    gem install bundler
> > +
> > +    bundle install
> > +
> > +    build.netstd.cmd
> > +test: off
> > +on_finish:
> > +    - ps: $root = Resolve-Path .\artifacts; [IO.Directory]::GetFiles($root.Path, '*.*', 'AllDirectories') | % { Push-AppveyorArtifact $_ -FileName $_.Substring($root.Path.Length + 1) -DeploymentName to-publish }
>
> The on_finish ensures that the entire artifacts directory is published
> even on a failed build. Without this, we get no logs for failed builds.
>
> Which logs are you talking about? AppVeyor always saves the full console
> output
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/FakeItEasy/FakeItEasy/pull/821/files/caa65210041c6c1c57f845a4783e5f1fc3cedc00#r75481834>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AApXSNSbIvyc5ElxD3XC7XGb6zfAlCrPks5qhbDegaJpZM4JcE68>
> .
>